scaper/core.py

```diff
@@ -1806,8 +1810,13 @@ def _generate_audio(self, audio_path, ann, reverb=None,
                             input_array=event_audio,
                             sample_rate_in=event_sr
                         )
+                        event_audio = event_audio.reshape(-1, self.n_channels)
+
                         soundfile.write(
-                            tmpfiles_internal[-1].name, event_audio.T, self.sr)
+                            tmpfiles_internal[-1].name, 
+                            event_audio.mean(axis=-1, keepdims=True), 
```

Comment:
This is just to match the loudness of the two-channel case vs the single-channel. Very much a hack.